Library/Services/Implementation/NfieldDomainEmailSettingsService.cs

```diff
@@ -55,11 +56,9 @@ public void InitializeNfieldConnection(INfieldConnectionClient connection)
             ConnectionClient = connection;
         }
 
-        private string DomainEmailSettingsUrl()
+        private Uri DomainEmailSettingsUrl()
         {
-            var result = new StringBuilder(ConnectionClient.NfieldServerUri.AbsoluteUri);
-            result.AppendFormat(CultureInfo.InvariantCulture, @"EmailSettings");
```

Comment:
weird that it was doing `AppendFormat` when there was no format :-/

Reply:
copy/paste from somewhere else i assume.